### PR TITLE
Doc/566/document digital asset

### LINKED ISSUE #566: Document the digital asset json template
to be done after #564

### PR DESCRIPTION
**Pull request by r-marques**
resolves #566 
to be merged after #599 

This documents the new model for digital assets. I will be add more documentation and code examples after implementing divisible assets

### CODE REVIEW

**Comment by ttmc** [comment]
The documentation pages with example output will have to be updated but I created a new issue for that. It can probably be automated somewhat. See issue #607

File: docs/source/topic-guides/models.md [hunk 1]
```diff
@@ -64,9 +64,10 @@ Assets can be mutable (changeable) or immutable. To change a mutable asset, you
         "conditions": ["<list of conditions>"],
         "operation": "<string>",
         "timestamp": "<timestamp from client>",
-        "data": {
+        "asset": "<digital asset description (explained in the next section)>",
+        "metadata": {
             "uuid": "<uuid4>",
```
Comment (ttmc) [inline docs/source/topic-guides/models.md]: Should be `"id": "<uuid>"`?

File: docs/source/topic-guides/models.md [hunk 2]
```diff
@@ -85,9 +86,10 @@ Here's some explanation of the contents of a transaction:
     See [Conditions and Fulfillments](#conditions-and-fulfillments) below.
     - `operation`: String representation of the operation being performed (currently either "CREATE" or "TRANSFER"). It determines how the transaction should be validated.
     - `timestamp`: The Unix time when the transaction was created. It's provided by the client. See [the section on timestamps](timestamps.html).
-    - `data`:
+	- `asset`: Definition of the digital asset. See next section.
+    - `metadata`:
         - `uuid`: UUID version 4 (random) converted to a string of hex digits in standard form.
```
Comment (ttmc) [inline docs/source/topic-guides/models.md]: Should this be:
```
- `id`: UUID version 4...
```

**Comment by ttmc** [comment]
The diagram in the section:

https://bigchaindb.readthedocs.io/en/latest/topic-guides/models.html#transaction-concepts

shows a transaction that transfers two different digital assets in the same transaction. I gather that will not be possible any more once PR #599 is merged. Is that right? Should we add a note in the docs that multi-asset transactions will be impossible for a while?

**Comment by r-marques** [comment]
I am not sure if there is a use case for multi asset transactions.
Anyway the diagram is still true for divisible assets, you may require multiple inputs from different transactions to get the desired amount for the output

**Comment by ttmc** [comment]
Suppose Joe bought a kettle and a toaster at the kitchen store, and the store would like to transfer both assets to Joe. The store could do that using two separate transfer transactions, and for two items that's not so bad, but what if Joe bought 40 items?

I'm not saying multi-asset transactions are a necessity, just nice to have. Maybe they could be a premium feature?

**Comment by ttmc** [comment]
:+1:

**Comment by r-marques** [comment]
deprecated in favor of #680

**Comment by r-marques** [comment]
closed wrong pr

**Comment by r-marques** [comment]
closed in favor of #688

**Pull request closed without merging**